### PR TITLE
Task-104-Getting-courses-for-carousel

### PR DESCRIPTION
**Pull request by melonholy**

### CODE REVIEW

File: src/components/couresCarousel/index.js [hunk 1]
```diff
@@ -3,6 +3,8 @@ import React, { Component } from 'react';
 import AliceCarousel from 'react-alice-carousel';
 import { IoIosArrowBack, IoIosArrowForward } from 'react-icons/io';
 
+import { getCoursesForCarousel } from '../../services/courses';
+
 import 'react-alice-carousel/lib/alice-carousel.css';
 import styles from './styles.module.scss';
```
Comment (universezzz) [inline src/components/couresCarousel/index.js]: Pls, move variable 'responsive' to the folder 'constants'

File: src/components/couresCarousel/index.js [hunk 2]
```diff
@@ -25,22 +27,30 @@ const responsive = {
 };
 
 class CoursesCarousel extends Component {
-  state = {};
+  state = {
+    courses: [],
+  };
+
+  componentDidMount() {
+    getCoursesForCarousel().then(courses => {
```
Comment (universezzz) [inline src/components/couresCarousel/index.js]: rewrite to async/await syntax pls

**Review by universezzz — CHANGES_REQUESTED** [review]

File: src/components/couresCarousel/index.js (hunk 2 above)
Comment (OlegPanasyuk) [reply src/components/couresCarousel/index.js]: And add error processing by try/catch

**Review by OlegPanasyuk — CHANGES_REQUESTED** [review]

File: src/services/courses.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+const config = {
+  headers: {
+    'Content-Type': 'application/json',
+  },
+};
+
+export const getCoursesForCarousel = async () => {
+  const { data } = await httpService.get({
+    url: links.corsesCarouselRoute,
```
Comment (universezzz) [inline src/services/courses.js]: courses

**Review by universezzz — APPROVED** [review]

**Review by OlegPanasyuk — APPROVED** [review]

**Pull request merged**